src/clandinin_lab_to_nwb/brezovec/brezovecimagingextractor.py

```diff
@@ -194,11 +189,13 @@ def get_timestamps(self) -> np.ndarray:
         timestamps = [absolute_times[t] for t in np.arange(0, len(absolute_times), self._num_planes_per_channel_stream)]
         return np.array(timestamps)
 
+    # Since we define one TwoPhotonSeries per channel, here it should return the name of the single channel
```

Comment:
Since we define one TwoPhotonSeries per channel, here it should return the name of the single channel: is that correct?

Reply:
Yes, I think you are correct.